### DISCUSSION TITLE
Not able to see any cameras at all after installation

### DISCUSSION THREAD

**Discussion by joe2k1** [mrlt8/docker-wyze-bridge#440] [Q&A] [answered]
Hello! 

Thank you for this amazing development. I cannot get this working for some reason. I am trying to set it up using my synology with docker. I have have read over the read me at least 20 times. I setup the username/password, added in a TZ plus the NET MODE is set to LAN. I have added the 3 ports (local and container are the same plus TCP only) that need to be redirected - 8888, 1935 and 8554. The log file shows the green check mark beside the cams. Although with the RTSP entries. It would appear it is working. I have a mix of v2 and v3s. However I do get this warning and I assume its a warning so it can ignored but thought I would mention it. Warning frame rate not available yet. It says starting all cameras! I am unable to view the URLs in either VLC or a web browser on a tablet. My local network is setup using an Eero mesh router. Http://ipaddressofcam:8888/name-of-cam is not working. I also noticed the new feature web ui is not working either - 172.17.0.3:5000 gives a cannot open, server not responding error message. Any help would be greatly appreciated. 

Also, I am running DSM version - DSM 7.1-42661 Update 2. I do get a warning about not running in a production environment.

**Comment by mrlt8** [comment, marked as answer]
I'm not very familiar with synology, but you'll need to expose port 5000 in your container to access the web-ui. 

You also need to use the IP of the bridge/host where the container is running to access the streams instead of the camera IP.

**Reply by joe2k1** [reply]
![6C4536D2-3A6A-4660-9B4B-22B1271B3892](https://user-images.githubusercontent.com/52223386/174698985-b4d9983e-d804-4af0-acef-d21aa87a3222.jpeg)

Wow thanks for the prompt reply. I’m afraid that the Synology web interface also runs on port 5000. Is it possible to change that port 5000 to something else? I’ve uploaded a screenshot (of the port settings) to help others with this issue. 

Edit: I thought I would try something based on your comments, It seems I can re-map the port but I get a blank webpage. This is progress as its not an error message. 

![466F7A71-07FD-4169-A747-24E2220DE2C9](https://user-images.githubusercontent.com/52223386/174699686-66ad02bb-9666-4ad1-ba3a-c6746f0f9ca4.jpeg)

I am experimenting, I see some entries in the log file which mention the wyze bridge, but I think the error message is cut off. It says “WyzeBridge” 172.17.0.1 DateTimeStamp “Get /name-of-cam HTTP/1.0” 404- ? Any ideas?

Edit: I tried adjusting the settings again and it seems to be working. I can open my web browser to see the cams using port 8888 and the name of the cam. Ie


Ipaddress of wyze bridge/docker:8888/name-of-cam

I mapped the port 5000 to port 7000 and it works, i can see all my cams.

I would love to know these setup doesn’t work with the Synology Surveillance software. If I pick streaming web or RTSP it needs to authenticate. Is there is a default username/password to pass on? 

Thanks again!

**Reply by hellboyfishman** [reply]
I have a synology so I gave it a quick go using Surveillance station.   

I was able to get it to work just fine by setting the brand to user defined and type to Streaming - RTSP.   Then you just need to use your synologyip:8554/name-of-camera and it will let you add it.   I didn't have to pass a username or password, just the ip:port/path.

**Reply by joe2k1** [reply]
> I have a synology so I gave it a quick go using Surveillance station.
> 
> I was able to get it to work just fine by setting the brand to user defined and type to Streaming - RTSP. Then you just need to use your synologyip:8554/name-of-camera and it will let you add it. I didn't have to pass a username or password, just the ip:port/path.

Thank you! This was my issue! It’s working now. The Surveillance Station is fantastic.
